tools/docker/Dockerfile

```diff
@@ -1,189 +1,177 @@
+# syntax=docker/dockerfile:1
```

Comment:
What do you think of adding a new set up section in the instructions: https://github.com/recommenders-team/recommenders/blob/main/SETUP.md

We could explain how to install docker in a command line and how to do it with VSCode. 

I'm particularly interested in the VSCode one.

What do you think?

Reply:
Sure, I will.